infra/helper_test.py

```diff
@@ -137,10 +143,19 @@ def test_generate_external_project(self):
     helper._generate_impl(
         helper.Project('/newfakeproject/',
                        is_external=True,
-                       build_integration_path=build_integration_path))
+                       build_integration_path=build_integration_path),
+        self.PROJECT_LANGUAGE)
     self._verify_templated_files(templates.EXTERNAL_TEMPLATES,
                                  build_integration_path)
 
+  def test_generate_swift_project(self):
+    """Tests that the swift project uses the correct base image."""
+    swift = "-swift"
```

Comment:
no need for the variable. Just use the strings directly as needed. Also this should just be `'swift'` since we already prepend '-' in `_generate_impl` right? 

I think that might also be why the test is failing. _generate_impl prepends a '-' so the generated result is 'base-builder--swift'. Bur our test comparison expects 'base-builder-swift'. 

```
=================================== FAILURES ===================================
_________________ GenerateImplTest.test_generate_swift_project _________________
[gw1] linux -- Python 3.8.11 /opt/hostedtoolcache/Python/3.8.11/x64/bin/python

self = <helper_test.GenerateImplTest testMethod=test_generate_swift_project>

    def test_generate_swift_project(self):
      """Tests that the swift project uses the correct base image."""
      swift = "-swift"
      helper._generate_impl(helper.Project(self.PROJECT_NAME), swift)
>     self._verify_templated_files(
          templates.TEMPLATES,
          os.path.join(helper.OSS_FUZZ_DIR, 'projects', self.PROJECT_NAME), swift)

infra/helper_test.py:155: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
infra/helper_test.py:128: in _verify_templated_files
    self.assertEqual(contents, template % template_args)
E   AssertionError: '# Co[704 chars]ilder\nRUN apt-get update && apt-get install -[160 chars]C/\n' != '# Co[704 chars]ilder-swift\nRUN apt-get update && apt-get ins[166 chars]C/\n'
E   Diff is 1034 characters long. Set self.maxDiff to None to see it.```